_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -47906,11 +47906,11 @@
 /obj/structure/window/reinforced,
 /obj/effect/decal/cleanable/cobweb/cobweb2,
 /obj/structure/showcase{
```

Comment:
@KorPhaeron It's also used on Delta. Make it a subtype instead and hit it there too.